libraries/AP_RCProtocol/AP_RCProtocol.h

```diff
@@ -110,6 +120,9 @@ class AP_RCProtocol {
         uint32_t last_baud_change_ms;
         enum config_phase phase;
     } added;
+
+    // allowed RC per
```

Comment:
This comment might need a few extra words added to the

Reply:
fixed :)